packages/backend-auth/src/factory.test.ts

```diff
@@ -26,8 +26,8 @@ import {
 
 const createStackAndSetContext = (): Stack => {
   const app = new App();
-  app.node.setContext('branch-name', 'testEnvName');
-  app.node.setContext('backend-id', 'testBackendId');
+  app.node.setContext(CDKContextKey.BACKEND_INSTANCE, 'testEnvName');
+  app.node.setContext(CDKContextKey.BACKEND_NAMESPACE, 'testBackendId');
```

Comment:
I'm on the fence about this. theoretically we should be able to change these names and as long as all the packages are on compatible versions, synthesis would continue to work uninterrupted. In practice though, it might be good to be alerted that you are making a potentially dangerous change.

Reply:
Updated to use the literal strings in test files.